main.cxx

```diff
@@ -49,6 +48,11 @@ int main( int argc, char *argv[] )
   mkdir(path_option[1].c_str(), 0755);
   fuse_argv[2] = strdup(path_option[1].c_str());
 
+  if(!check_permissions(fuse_argv[2])){
+    fastlog(ERROR, "Fatal error occurred. Aborting!");
```

Comment:
We can have a bit more descriptive error message here to show that permission is denied.

Reply:
No: the detailed errors are printed by the specific tests. Otherwise every time we add a test we have to bring out of file the error message.